test/data/carol

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
+MC4CAQAwBQYDK2VwBCIEICmtWWk/6UydYjr7tmVUtPa7JIxHdhaJraSHXr2pSECu
+-----END PRIVATE KEY-----
```

Comment:
ah, I see now. This is an ed25519 pem?

Reply:
Yes correct, generated via: ` openssl genpkey -algorithm ed25519 -outform PEM -out carol`